### ISSUE TITLE
how to convert dae file to obj+mtl+png?

### ISSUE THREAD

**Issue by kaixin-bai** [mikedh/trimesh#886]
how to convert dae file to a obj file with texture?

**Comment by mikedh** [comment]
It really should be as simple as:
```
    s = trimesh.load('models/duck.zae')
    s.export('./duck.obj')
```

However there were a number of small bugs preventing this from working. Should be fixed in #891

**Comment by mikedh** [comment]
...also assuming you have pycollada and PIL installed :smile:
